.github/gitops-action/action.yml

```diff
@@ -18,7 +18,9 @@ runs:
     - name: Install fleetctl
       shell: bash
       working-directory: ${{ inputs.working-directory }}
-      run: npm install -g fleetctl
+      run: |
+        FLEET_VERSION="$(curl "$FLEET_URL/api/v1/fleet/version" --header "Authorization: Bearer $FLEET_API_TOKEN" --silent | jq --raw-output '.version')"
+        npm install -g fleetctl@$FLEET_VERSION
```

Comment:
We should check if `FLEET_VERSION` is not empty (e.g. if the curl request fails for some reason it will fail silently).

Reply:
Good call - I'm thinking if the server fails to return a version we should just default to the latest version?

Reply:
Yes, I'm thinking log some warning that version could not be extracted, and use latest.
But let's wait for some feedback from @getvictor and @noahtalerman.

Reply:
I agree: log and install latest. Otherwise looks good.

Reply:
I added the `--fail` flag to the `curl` command so it should catch _most_ (all?) errors 🤞

Time for another review :pray: